src/actions/generate.ts

```diff
@@ -0,0 +1,21 @@
+import { ChainId } from "@iov/bcp";
+import { Bip39, Random } from "@iov/crypto";
+import { UserProfile } from "@iov/keycontrol";
+
+import { codecFromString } from "../codec";
+import { setSecretAndCreateIdentities } from "../profile";
+
+export async function generate(args: ReadonlyArray<string>): Promise<void> {
+  if (args.length < 2) {
+    throw Error(`Not enough arguments for action 'generate'. See README for arguments.`);
```

Comment:
`help` is probably more useful for a user at this point.

Reply:
Yeah, I was thinking about that too. Will update